apps/hash-ai-worker-ts/src/activities/flow-activities/shared/get-answers-from-human.ts

```diff
@@ -0,0 +1,27 @@
+import { generateUuid } from "@local/hash-isomorphic-utils/generate-uuid";
+import { Context } from "@temporalio/activity";
+
+import { requestExternalInput } from "../../shared/request-external-input";
+
+export const getAnswersFromHuman = async (
+  questions: string[],
+): Promise<string> => {
+  const {
+    data: { answers },
+  } = await requestExternalInput({
+    requestId: generateUuid(),
+    stepId: Context.current().info.activityId,
```

Comment:
The `stepId` can be obtained from `getFlowContext` now.

Reply:
Done in https://github.com/hashintel/hash/pull/4461/commits/0cce668ff26764f8536f632d796bf73a594e1b68